tests/test_rootfs.py

```diff
@@ -104,3 +116,12 @@ def test_expected_files_ubifs(self, bitbake_path, bitbake_variables, latest_ubif
             with open(path) as fd:
                 data = fd.read()
             TestRootfs.verify_fstab(data)
+
+    @pytest.mark.only_with_distro_feature('mender-convert')
+    @pytest.mark.min_mender_version("1.0.0")
+    def test_unconfigured_image(self, latest_rootfs):
+        """Test that images from mender-convert are unconfigured. We want
+        `mender setup` to be the configuration mechanism there."""
+        output = subprocess.check_output(["debugfs", "-R", "ls -l -p /etc/mender",
+                                          latest_rootfs]).decode()
+        assert "mender.conf" not in output
```

Comment:
Thinking now about it, `mender setup` will not configure the A/B partition configuration. So there must be a `mender.conf` somewhere in the device, right? Is it created in `/data/mender/mender.conf`? I think we should check that it exists and that contains only the RootfsPartA/RootfsPartB values.

Reply:
This is [already tested elsewhere](https://github.com/mendersoftware/mender-image-tests/blob/master/tests/test_part_image.py#L296-L322).

Reply:
You are right!